### PR TITLE
vt_best_practices: cover the qemu-ksm package

### PR DESCRIPTION
**Pull request by dfaggioli**
### Description

Mention, when talking about KSM, that the qemu-ksm package is now
available and that, if installed, it provides a service file that can be
used to start and stop KSM itself.

### Are there any relevant issues/feature requests?

* jsc#SLE-11692

### To which product versions do the changes apply?

| Code 15     | Backport Done
| ----------  | ---------- 
| [x] 15 SP3  | (`master` only)   
| [ ] 15 SP2  | [ ] 
| [ ] 15 SP1  | [ ] 
| [ ] 15 SP0  | [ ] 

| Code 12     | Backport Done
| ----------  | ---------- 
| [ ] 12 SP5  | [ ] 
| [ ] 12 SP4  | [ ] 
| [ ] 12 SP3  | [ ] 
| [ ] 12 SP2  | [ ]

### CODE REVIEW

File: xml/vt_best_practices.xml [hunk 1]
```diff
@@ -479,7 +479,12 @@ always madvise [never]</screen>
      consumption on the &vmhost; by sharing data &vmguest;s have in common. The
      KSM daemon <systemitem class="daemon">ksmd</systemitem> periodically scans
      user memory looking for pages of identical content which can be replaced
-     by a single write-protected page. To enable KSM, run:
+     by a single write-protected page. To enable KSM, start the relative service:
```
Comment (cjschroder) [inline xml/vt_best_practices.xml:482]: "relative" should be "ksm service"

File: xml/vt_best_practices.xml [hunk 2]
```diff
@@ -479,7 +479,12 @@ always madvise [never]</screen>
...
+    </para>
+<screen>&prompt.user;&sudo; systemctl enable --now ksm.service</screen>
+    <para>
+     Which will be available if the package <package>qemu-ksm</package>
+     is installed. Alternatively, it can also be started by doing:
```
Comment (cjschroder) [inline xml/vt_best_practices.xml:487]: I suggest replacing "doing" with "running the following command:" It is better to be explicit.

**Review by cjschroder — CHANGES_REQUESTED** [review]
Thank you @dfaggioli, appreciate very much your PR! This is good work dealing with our complex toolchain :) I suggested a couple of small corrections.  cheers and thanks!

File: xml/vt_best_practices.xml (hunk 1 above)
Comment (dfaggioli) [reply xml/vt_best_practices.xml:482]: Thanks for the comments! I see your point, and agree that "relative" is bad. However, re-reading the sentence, I don't think I really like it. I think I'm turning it into: "To enable the KSM service and start it, run the following command:"

File: xml/vt_best_practices.xml (hunk 2 above)
Comment (dfaggioli) [reply xml/vt_best_practices.xml:487]: Ok, done (although slightly different, as I did use the sentence "run the following command" above)

I will update the pull request

**Comment by cjschroder** [comment]
Luvverly, thanks! Merging.

**Pull request merged**